packages/block-editor/src/components/block-canvas/style.scss

```diff
@@ -1,7 +1,7 @@
 iframe[name="editor-canvas"] {
+	box-sizing: border-box;
 	width: 100%;
 	height: 100%;
 	display: block;
-	background-color: $gray-300;
```

Comment:
So this background is not needed at all, even on the post editor?

Reply:
I was afraid that this would break Safari, but this is what I see:

<img width="1156" alt="Screenshot 2024-06-04 at 11 41 57" src="https://github.com/WordPress/gutenberg/assets/3593343/abf0fb36-e1c4-4dcf-9917-b8633caa76bf">

Reply:
In my testing, these contexts inherit from the other class.

What do you say, good to land?